Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1234,6 +1298,29 @@ def search_file_command(file):
     get_indicator_outputs(indicator_type, indicator_details, 'SHA256')
 
 
+def get_export_list_command(args):
+    data = {
+        'label': args.get('label'),
+        'panosFormatted': True,
```

Comment:
add an explanation for this and why don;t you need an api key

Reply:
No api key needed here as it is added in the http_request command.
added explanation

Reply:
so maybe we should remove this?

Reply:
when I tried to run without this line it raised an error... I'll try again...